dkan.profile

```diff
@@ -411,3 +411,109 @@ function dkan_set_roleassign_roles(&$context) {
   }
   variable_set('roleassign_roles', $roleassign_roles);
 }
+
+/**
+ * Configures BUEditor and markdown test format.
+ */
+function dkan_bueditor_markdown_install() {
```

Comment:
@acouch this function doesn't make sense inside the profile.  It only gets called by sites on updates not on first time installation.